### ISSUE TITLE
DTS-HD MA misdetected as lossy DTS

### ISSUE THREAD

**Issue by vstax** [MediaArea/MediaInfo#738]
Audiotrack from "Tammy and the T-Rex" movie (4K UHD) which is DTS HD-MA - correctly detected by AVR, ffmpeg:
```
Input #0, matroska,webm, from 'tammy-dtshd.mka':
  Metadata:
    encoder         : libebml v1.4.4 + libmatroska v1.7.1
    creation_time   : 2023-05-19T14:59:48.000000Z
  Duration: 00:00:03.01, start: 0.000000, bitrate: 2336 kb/s
  Stream #0:0(eng): Audio: dts (DTS-HD MA), 96000 Hz, stereo, s32p (24 bit) (default)
    Metadata:
      BPS             : 2319643
      DURATION        : 00:00:03.007664530
      NUMBER_OF_FRAMES: 282
      NUMBER_OF_BYTES : 871896
      SOURCE_ID       : 001100
      _STATISTICS_WRITING_APP: mkvmerge v76.0 ('Celebration') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2023-05-19 14:59:48
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES SOURCE_ID
```

and MPC-BE's internal mediainfo as DTS-HD MA 96 KHz, is incorrectly detected as plain lossy DTS with 48Khz by mediainfo 23.04:
```
{
"creatingLibrary": {
"name": "MediaInfoLib",
"version": "23.04",
"url": "https://mediaarea.net/MediaInfo"
},
"media": {
"@ref": "tammy-dtshd.mka",
"track": [
{
"@type": "General",
"UniqueID": "214472875254271280325235789957098317545",
"AudioCount": "1",
"FileExtension": "mka",
"Format": "Matroska",
"Format_Version": "4",
"FileSize": "878517",
"Duration": "3.008",
"OverallBitRate_Mode": "CBR",
"OverallBitRate": "2336481",
"StreamSize": "6621",
"IsStreamable": "Yes",
"Encoded_Date": "2023-05-19 14:59:48 UTC",
"File_Modified_Date": "2023-05-19 14:59:48 UTC",
"File_Modified_Date_Local": "2023-05-19 21:59:48",
"Encoded_Application": "mkvmerge v76.0 ('Celebration') 64-bit",
"Encoded_Library": "libebml v1.4.4 + libmatroska v1.7.1"
},
{
"@type": "Audio",
"StreamOrder": "0",
"ID": "1",
"OriginalSourceMedium_ID": "4352",
"UniqueID": "6515374977116105612",
"Format": "DTS",
"Format_Settings_Mode": "16",
"Format_Settings_Endianness": "Big",
"CodecID": "A_DTS",
"Duration": "3.007664530",
"BitRate_Mode": "CBR",
"BitRate": "1509000",
"Channels": "2",
"ChannelPositions": "Front: L R",
"ChannelLayout": "L R",
"SamplesPerFrame": "512",
"SamplingRate": "48000",
"SamplingCount": "144368",
"FrameRate": "93.750",
"FrameCount": "282",
"BitDepth": "24",
"Compression_Mode": "Lossy",
"Delay": "0.000",
"Delay_Source": "Container",
"StreamSize": "871896",
"Language": "en",
"Default": "Yes",
"Forced": "No",
"extra": {
"OriginalSourceMedium": "Blu-ray"
}
}
]
}
}
```

I've uploaded sample to https://www.mediafire.com/file/b1fxtkdyx648sds/tammy-dtshd.mka/file

**Comment by JeromeMartinez** [comment]
Tested with v23.06 and it seems already fixed (displayed as lossless), you have v23.04, please test with 23.06

**Comment by vstax** [comment]
Right, it works correctly now. Sorry, I just didn't realize that 23.06 was released!
